heroku_connect/db/models/base.py

```diff
@@ -64,7 +64,7 @@ class MyCustomObject(hc_models.HerokuConnectModel):
 
         A model mixin must inherit from :class:`Meta.managed<django.db.models.Model>`
         not `.HerokuConnectModel`. Only the final (not abstract) models should inherit
-        from `.HerokuConnectModel` otherwise build in fields will clash.
+        from `.HerokuConnectModel` otherwise build-in fields will clash.
```

Comment:
Although a small change - I wanted to imply that its `built-in` . 
https://docs.djangoproject.com/en/2.0/howto/custom-model-fields/#emulating-built-in-field-types

Reply:
I fail to see how this article is related. I am talking about the fields build into `HerokuConnectModel`.

Reply:
Yes, I understand. The link was just to show the heading, which reads `built-in fields`  and not `build-in`.